<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconstrucci√≥n en 3D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="BotonPlay">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Tomar Fotos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonStop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Detener Capturas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonEjecutar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejecutar</string>
    </property>
   </widget>
   <widget class="MLabel" name="LabelCamara">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>480</width>
      <height>360</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="coor">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x=0 y =0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MLabel</class>
   <extends>QLabel</extends>
   <header>mlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
